src/main/java/com/google/api/codegen/gapic/MainGapicProviderFactory.java

```diff
@@ -416,7 +416,7 @@
 
         providers.add(mainProvider);
         providers.add(clientConfigProvider);
-        providers.add(metadataProvider);
+        //        providers.add(metadataProvider);
```

Comment:
Let's not comment out code, unless there's a TODO comment explaining when we will uncomment it. Otherwise, just delete it.